src/tests/test_updater.py

```diff
@@ -52,7 +52,7 @@ def test_new_organism(self, multi_dbs):
             assert dataset.dataset_source.type == "core"
             assert dataset.dataset_type.name == "genebuild"
 
-    #
+    # Just this one!.
```

Comment:
not sure about the meaning of this comment :-)

Reply:
Shouldn't have been left in.